CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## UNRELEASED
+
+* Import now requires the edit permission.
```

Comment:
Should be both edit and create permission because a CSV file import can potentially do both.